eufy_security/camera.py

```diff
@@ -85,3 +103,29 @@ def station_serial(self) -> str:
     async def async_update(self) -> None:
         """Get the latest values for the camera's properties."""
         await self._api.async_update_device_info()
+
+    async def async_set_params(self, params: dict) -> None:
```

Comment:
The convention in this project is to put the methods in alphabetical order.

Reply:
Additionally, will a client ever call this method directly? If `async_start_detection` and `async_stop_detection` are the public-facing methods, let's rename this to `_async_set_params`.

Reply:
Start/stop detection method are intended for the HA integration. BTW, I extracted your PR [here](https://github.com/nonsleepr/ha-eufy-security).
I would think the users of the library might want to make changes in batches and would prefer to keep it public.